### DISCUSSION TITLE
How do you dockerize a WordPlate setup?

### DISCUSSION THREAD

**Discussion by wbern** [wordplate/wordplate#346] [Q&A]
I've been scouring the web and also been trying to set up a docker-compose.yml file for the database, php with php-fpm and nginx.

I can manage a basic setup, but I run into two issues.

* I get routed to /wordpress/wp-admin instead of just /wp-admin
* WordPress is unable to load the themes in the public directory, it seems to be expecting the themes to be in the public/wordpress directory.

Do you have any tips or examples of how people have set up wordplate with docker?

Many thanks in advance!

**Comment by vinkla** [comment]
> I get routed to /wordpress/wp-admin instead of just /wp-admin

This is the expected behaviour since we install WordPress as a dependency within the `public` directory.

> WordPress is unable to load the themes in the public directory, it seems to be expecting the themes to be in the public/wordpress directory.

I haven't experienced this in the past.

> Do you have any tips or examples of how people have set up wordplate with docker?

I haven't used WordPlate with Docker but it should be similar setup to projects such as Laravel Sail or Bedrock Docker.

**Comment by rydurham** [comment]
I have worked out a docker setup for WordPlate: https://github.com/stagerightlabs/wordplate-vite-docker
